package.json

```diff
@@ -18,6 +18,7 @@
 		"@typescript-eslint/eslint-plugin": "^5.14.0",
 		"@typescript-eslint/parser": "^5.14.0",
 		"cross-spawn": "^7.0.3",
+		"csv-stringify": "^6.0.5",
```

Comment:
We were using an old version of `csv-stringify` that we were actually getting as a transitive dependency on something else, so I added it as a direct dependency.
Approved 3PP: https://gus.lightning.force.com/lightning/r/ADM_Third_Party_Software__c/a0qEE0000002P7BYAU/view